R/get_mailchimp_data.R

```diff
@@ -1,21 +1,47 @@
 #' Get mailchimp data
 #' @param endpoint Name of target data to access under api.mailchimp.com
 #' e.g. "reports", "lists/members"
-#' @param date_since Filter data by date
+#' @param date_type Type of date_since argument. Can be "exact", "days", "weeks", "months" or "years".
+#' "exact" uses exact date like "2016-01-01".
+#' "days", "weeks", "months" or "years" uses a number and get data from that time ago.
```

Comment:
"from that time ago" does not sound like a common English phrase to me. Maybe "since that much time ago" is better.